pulser/simulation/simconfig.py

```diff
@@ -99,21 +99,29 @@ def __post_init__(self) -> None:
         self._check_spam_dict()
         self._calc_sigma_doppler()
 
-    def __str__(self) -> str:
-        lines = [
-            "Options:",
-            "----------",
-            "Noise types:           " + ", ".join(self.noise),
-            f"Spam dictionary:       {self.spam_dict}",
-            f"Temperature:           {self.temperature}K",
-            f"Number of runs:        {self.runs}",
-            f"Samples per runs:      {self.samples_per_run}",
-            f"Laser waist:           {self.laser_waist}μm",
-            f"Dephasing probability: {self.dephasing_prob}",
-            "Solver Options:",
-            f"{str(self.solver_options)[10:-1]}",
-        ]
-        return "\n".join(lines)
+    def __str__(self, solver_options: bool = False) -> str:
+        lines = ["Options: \n", "---------- \n"]
+        lines.append(f"Number of runs:        {self.runs} \n")
+        lines.append(f"Samples per run:       {self.samples_per_run} \n")
+        if self.noise:
+            lines.append(
+                "Noise types:           " + ", ".join(self.noise) + "\n"
+            )
+        if "SPAM" in self.noise:
+            lines.append(f"Spam dictionary:       {self.spam_dict} \n")
+        if "doppler" in self.noise:
+            lines.append(
+                f"Temperature:           {self.temperature*1.e6}µK \n"
+            )
+        if "amplitude" in self.noise:
+            lines.append(f"Laser waist:           {self.laser_waist}μm \n")
+        if "dephasing" in self.noise:
+            lines.append(f"Dephasing probability: {self.dephasing_prob} \n")
+        if solver_options:
+            lines.append(
+                "Solver Options: \n" + f"{str(self.solver_options)[10:-1]}"
+            )
+        return "".join(lines).rstrip()
```

Comment:
Thanks for the change! I think that you might be able to go back to the `"\n".join(...)` now